portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -3928,6 +3928,94 @@ definition {
 			fieldValue = "Checkbox 1");
 	}
 
+	@description = "This is a use case for LPS-125860."
+	@priority = "5"
+	test SubmitFormWithRequiredConfirmationTextField {
```

Comment:
@brunofariasdeo and @carolmariaabb, is this Poshi test covering a scenario not already covered by the unit tests? Is it a complementary part of the created unit tests?

Reply:
It is a complementary part to ensure that everything together (FE + BE) is working as expected.